packages/ounit-lwt/ounit-lwt.2.1.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvaini+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ounit"
+dev-repo: "git+https://github.com/gildor478/ounit.git"
+bug-reports: "https://github.com/gildor478/ounit/issues"
+doc: "https://gildor478.github.io/ounit/"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {build & >= "1.11.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11.0"}
```

As of few months, dune cannot be listed as build dependency due to a downgrading issue

Reply:
Fixed in the new release. Sorry for the many releases -- ounit is a revdep for a lot of packages and I want to be careful with this new release.

Closing the PR since the v2.1.2 has its own PR